OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameChatLogic.cs

```diff
@@ -69,6 +69,9 @@ public IngameChatLogic(Widget widget, OrderManager orderManager, World world, Mo
 			var teamMessage = modData.Translation.GetString(Team);
 			var allMessage = modData.Translation.GetString(All);
 
+			if (TextNotificationsManager.MutedPlayers.Count == 0)
```

Comment:
Is there a case where there will be non-zero players in the dictionary here?

Reply:
Yes, this is called every time you open the ingame menu and it has to be here as UI is one of the first things that's loaded into the game

Reply:
Ah, right.